docs/installation/ansible-galaxy.rst

```diff
@@ -0,0 +1,20 @@
+Ansigle Galaxy Installation
+===========================
+
+To have fully installable Pulp3 it is needed to:
+
+1. Ensure python3, ansible( >2.2), libselinux-python and virtualenv are installed
```

Comment:
Can you clarify which of these is needed on the control machine vs the target machines? ex. ansible on control, virtualenv on target